includes/class-amp-theme-support.php

```diff
@@ -1941,7 +1941,7 @@ public static function prepare_response( $response, $args = [] ) {
 		 * Abort if the response was not HTML. To be post-processed as an AMP page, the output-buffered document must
 		 * have the HTML mime type and it must start with <html> followed by <head> tag (with whitespace, doctype, and comments optionally interspersed).
 		 */
-		if ( Attribute::TYPE_HTML !== substr( AMP_HTTP::get_response_content_type(), 0, 9 ) || ! preg_match( '#^(?:<!.*?>|\s+)*<html.*?>(?:<!.*?>|\s+)*<head\b(.*?)>#is', $response ) ) {
+		if ( Attribute::TYPE_HTML !== substr( AMP_HTTP::get_response_content_type(), 0, 9 ) || ! preg_match( '#^(?:<!.*?>|\s+)*+<html.*?>(?:<!.*?>|\s+)*+<head\b(.*?)>#is', $response ) ) {
 			return $response;
 		}
```

Comment:
Yes, now that we have the `$did_template_action` check above, this condition should never evaluate to true. So I think the whole `if` statement can be removed now as being redundant.

Reply:
Yea that makes sense.

Reply:
Pushing that up…